heat/core/random.py

```diff
@@ -422,6 +427,43 @@ def randn(*args, dtype=types.float32, split=None, device=None, comm=None):
     return normal_tensor
 
 
+def random(
+    shape: Optional[Tuple[int]] = None,
+    dtype=types.float32,
+    split: Optional[int] = None,
+    device: Optional[str] = None,
+    comm=None,
+):
+    """
+    Random values in a given shape.
+    Create a :class:`~heat.core.dndarray.DNDarray`  of the given shape and populate it with random samples from a
+    uniform distribution over [0, 1).
+    Parameters
+    ----------
+    shape : Tuple[int]
+        The shape of the returned array, should all be positive. If no argument is given a single random sample is
+        generated.
+    dtype: Type[datatype], optional
+        The datatype of the returned values. Has to be one of
+        [:class:`~heat.core.types.float32, :class:`~heat.core.types.float64`].
+    split: int, optional
+        The axis along which the array is split and distributed, defaults to no distribution.
+    device : str, optional
+        Specifies the :class:`~heat.core.devices.Device`  the array shall be allocated on, defaults to globally
+        set default device.
+    comm: Communication, optional
+        Handle to the nodes holding distributed parts or copies of this array.
+    """
+    if not shape:
+        shape = (1,)
+    shape = stride_tricks.sanitize_shape(shape)
+    return rand(*shape, dtype=dtype, split=split, device=device, comm=comm)
+
+
+# aliases
+random_sample = ranf = sample = random
```

Comment:
where is `randf` from? i cant find that name in other packages

Reply:
I found `ranf` it as an alias in numpy.